generator/internal/sidekick/generate.go

```diff
@@ -24,6 +24,19 @@ import (
 	toml "github.com/pelletier/go-toml/v2"
 )
 
+var CmdGenerate = NewCommand(
```

Comment:
I would construct these within an `init` command:
https://go.dev/doc/effective_go#init

Reply:
I moved all commands to init functions within their source files. The only command I kept out of an init function was cmdSidekick, since that one needs to be  visible within the package, and I didn't see a reason to declare it outside the init function, and construct it within the init function. Let me know if you think that would make sense to change too.

Reply:
sgtm - we can always move things around as a follow up